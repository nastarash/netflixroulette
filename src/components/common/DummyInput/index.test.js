import React from 'react';
import { DummyInput } from './index';

describe('<DummyInput />', ()=> {

  test('should be rendered correctly', () => {
    const input = shallow(
      <DummyInput />
    );
    expect(input).toMatchSnapshot();
  });

});
