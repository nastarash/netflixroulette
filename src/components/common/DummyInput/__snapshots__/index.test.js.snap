// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DummyInput /> should be rendered correctly 1`] = `
<div
  className="input"
>
  <span
    className="input__icon"
  >
    â†µ
  </span>
  <input
    className="input__text"
    type="text"
  />
</div>
`;
